### PR TITLE
`MultiRepodata.Unique` implemented

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #26 
Implemented `MultiRepodata.Unique` and added simple test (will add more tests in the next PR). 
Here is repodata.json structure:
```
{
  "packages": {
    //tar.bz2 packages listed here//
  },
  "packages.conda": {
    //conda packages listed here//
  }
}
```
The order of the "packages" and "packages.conda" items in repodata.json is not specified in the documentation, thus there is no way to avoid writing "packages" and "packages.conda" in the different temp files.

### CODE REVIEW

File: src/main/java/com/artipie/conda/MultiRepodata.java [hunk 1]
```diff
@@ -32,11 +45,123 @@
...
+        // @checkstyle ExecutableStatementCountCheck (30 lines)
         @Override
-        public void merge(final Collection<InputStream> input, final OutputStream result) {
-            throw new NotImplementedException("Not implemented yet");
+        public void merge(final Collection<InputStream> inputs, final OutputStream result) {
+            final JsonFactory factory = new JsonFactory();
+            try {
+                final Path ftars = Files.createTempFile("tars", Unique.EXT);
+                final Path fcondas = Files.createTempFile("condas", Unique.EXT);
+                // @checkstyle NestedTryDepthCheck (20 lines)
```
Comment (g4s8) [inline src/main/java/com/artipie/conda/MultiRepodata.java:72]: Is it possible to avoid creating temporary files and using `intputs` and `result` directly?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova just one questions

File: src/main/java/com/artipie/conda/MultiRepodata.java (hunk 1 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/conda/MultiRepodata.java:72]: @g4s8 I'm not sure I understand the question... Inputs are used directly: I read them one by one, but I do not know the order of `packages` and `packages.conda` elements. If the order was strict, I could first read all the inputs and write `tar.bz2` items to the `result` directly until `packages.conda` is met and then continue reading inputs and write `packages.conda` elements. 
Check [readme](https://github.com/artipie/conda-adapter#repodata-file) for better understanding of the `repodata.json` structure.

**Review by g4s8 — APPROVED** [review]
@olenagerasimova ok, thanks

**Pull request merged**